### PR TITLE
 Agent-10: Adds openshift-install agent wait-for bootstrap-complete  subcommand

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to main and 4.12
- [Jira](https://issues.redhat.com/browse/AGENT-10)
- [Preview](https://52973--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html)
- @lranjbar ptal and review.

### CODE REVIEW

**Comment by Amrita42** [comment]
/label peer-review-needed

Note that content [step 1] is re-used from (https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal/installing-bare-metal-network-customizations.html#installation-installing-bare-metal_installing-bare-metal-network-customizations)

**Comment by stevsmit** [comment]
LGTM! Send it over to merge review when you're ready!

File: modules/installing-ocp-agent.adoc [hunk 1]
```diff
@@ -147,6 +147,30 @@ agent.iso  auth
 $ openshift-install agent create image
 ----
 
+. Optional:  To know  when the bootstrap node (** Node 0 **) reboots, run the following command:
```
Comment (bergerhoffer) [inline modules/installing-ocp-agent.adoc:150]: * "To know" doesn't sound right. I'd suggest something like "To determine" or another option instead.
* Usually bold is reserved for GUI items, so "Node 0" should likely use some other markup, if any at all. There also aren't that many other references to "node 0" in the docs, so I wonder if that's needed at all. The ones I do find have it lowercased too, just FYI.

File: modules/installing-ocp-agent.adoc [hunk 2]
```diff
@@ -147,6 +147,30 @@ agent.iso  auth
...
+.Example output
+[source,terminal]
+----
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.25.0 up
+INFO Waiting up to 30m0s for bootstrapping to complete...
+INFO It is now safe to remove the bootstrap resources
+----
++
+The command succeeds when the Kubernetes API server signals that it has been
```
Comment (bergerhoffer) [inline modules/installing-ocp-agent.adoc:171]: This sentence shouldn't be hard wrapped.

**Review by bergerhoffer — COMMENTED** [review]
Added a few items that you might want to check out, but I'm not going to hold up the merge over it.

Merging!

**Pull request merged**